numpy/conftest.py

```diff
@@ -16,6 +16,8 @@
 def pytest_configure(config):
     config.addinivalue_line("markers",
         "valgrind_error: Tests that are known to error under valgrind.")
+    config.addinivalue_line("markers",
+        "leaks_references: Tests that are known to leak references.")
```

Comment:
I do not mind if we add them or not. I did not actually use them yet, so a comment in the test will do (or the correct magic to actually manage to create such arrays, I am starting to feel we have been always supporting something that is exceedingly difficult to achieve.